pkg/piecestore/rpc/client/client.go

```diff
@@ -38,9 +38,9 @@ var (
 // PSClient is an interface describing the functions for interacting with piecestore nodes
 type PSClient interface {
 	Meta(ctx context.Context, id PieceID) (*pb.PieceSummary, error)
-	Put(ctx context.Context, id PieceID, data io.Reader, ttl time.Time, ba *pb.PayerBandwidthAllocation) error
-	Get(ctx context.Context, id PieceID, size int64, ba *pb.PayerBandwidthAllocation) (ranger.Ranger, error)
-	Delete(ctx context.Context, pieceID PieceID) error
+	Put(ctx context.Context, id PieceID, data io.Reader, ttl time.Time, ba *pb.PayerBandwidthAllocation, sm *pb.SignedMessage) error
```

Comment:
Let's pick a name for the `sm` param that communicates better the purpose of the signed message. The purpose is to authorize that the client is allowed to execute the respective operation, isn't it? In such case a name like `sanction`, `permit`, or `authorization` would fit.

Reply:
done, i like `authorization` :)